### PR TITLE
operator SFS upgrade notes

### PR DESCRIPTION
**Pull request by oraNod**

### CODE REVIEW

File: documentation/asciidoc/topics/ref_upgrades.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+====
+{ispn_operator} requires the Operator Lifecycle Manager to perform cluster upgrades.
+====
+
+[discrete]
+== Upgrade notes
+
+If you upgrade {brandname} clusters manually, you should upgrade to the latest {brandname} 12.x version if you have already upgraded {ispn_operator} from 2.0.x to 2.1.x, which corresponds to {brandname} Server 11.x to 12.x.
+
+You should perform this upgrade as soon as possible to avoid potential data loss that can occur after upgrading to {brandname} 12.1.0 through to 12.1.4 from an earlier {brandname} version.
```
Comment (ryanemerson) [inline documentation/asciidoc/topics/ref_upgrades.adoc:18]: We should link the upstream server Jira here: https://issues.redhat.com/browse/ISPN-13116

File: documentation/asciidoc/topics/ref_upgrades.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+[NOTE]
+====
+{ispn_operator} requires the Operator Lifecycle Manager to perform cluster upgrades.
+====
+
+[discrete]
+== Upgrade notes
+
+If you upgrade {brandname} clusters manually, you should upgrade to the latest {brandname} 12.x version if you have already upgraded {ispn_operator} from 2.0.x to 2.1.x, which corresponds to {brandname} Server 11.x to 12.x.
```
Comment (ryanemerson) [inline documentation/asciidoc/topics/ref_upgrades.adoc:16]: Maybe we should reword this to:

"If you upgrade {brandname} clusters manually and have upgraded {ispn_operator} channel from 2.0.x to 2.1.x you should approve the latest {brandname} 12.x version as soon as possible. This is in order to mitigate the impact of ISPN-13116 and prevent data loss from occurring."

Comment (oraNod) [reply documentation/asciidoc/topics/ref_upgrades.adoc]: thanks @ryanemerson I've updated this but slightly reworded

**Comment by ryanemerson** [comment]
Thanks @oraNod. I'll add the commit to my PR on the 2.1.x branch
